packages/core/cli.js

```diff
@@ -63,11 +63,22 @@ const command = getCommand({
   options: {},
   noAliases: false
 });
-const options = prepareOptions({
-  command,
-  inputStrings,
-  options: {}
-});
+const options = (function () {
```

Comment:
Does this have to be an [IFFE](https://developer.mozilla.org/en-US/docs/Glossary/IIFE)?

Reply:
Yeah I don't know why this should be constructed inside a function like this. It seems like it should read something like
```
const command = getCommand(....
if (command === null) {
  // deal with error stuff
}
const options = prepareOptions(...
```

Or maybe you have a reason for writing it the way you did?

Reply:
agreed. I changed it.